### DISCUSSION TITLE
how to use new PDK in OpenROAD-flow-scripts?

### DISCUSSION THREAD

**Discussion by 706697916** [The-OpenROAD-Project/OpenROAD-flow-scripts#1049] [General] [answered]
OpenROAD supports Open PDK and some commercial PDK. That is great! 
Now I want to use OpenROAD in new PDK. How to develop it? Is there some tutorial to help me?

**Comment by vijayank88** [comment, marked as answer]
Refer this document: https://openroad-flow-scripts.readthedocs.io/en/latest/contrib/PlatformBringUp.html

**Reply by 706697916** [reply]
Thank you！
